### PR TITLE
Refine README.md to update futures version

### PR DESCRIPTION
**Pull request by JmPotato**
Signed-off-by: JmPotato <ghzpotato@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,8 +14,8 @@ _It is still a work in progress._
   scheduling algorithms like [multi-level feedback queue][MLFQ].
 
 * **Generic**: Yatp is easily adapted to various tasks. Simple callbacks,
-  [futures 0.1] and [std Future] are built-in supported.
+  [futures 0.3] and [std Future] are built-in supported.
```
Comment (BusyJay) [inline README.md]: Perhaps we can remove it considering std future has been release for such long time. /cc @sticnarf

Comment (sticnarf) [reply README.md]: Agree

Comment (JmPotato) [reply README.md]: Addressed. PTAL.

**Review by BusyJay — DISMISSED** [review]

**Review by sticnarf — DISMISSED** [review]

**Comment by JmPotato** [comment]
Seems that the CI stucked.

**Comment by BusyJay** [comment]
Perhaps we should switch to github actions 😢

**Comment by JmPotato** [comment]
> Perhaps we should switch to github actions 😢

I'd like to help. Working on it now.

**Review by BusyJay — APPROVED** [review]
Thanks!

**Pull request merged**